native/src/Navigator.tsx

```diff
@@ -93,6 +93,8 @@ const Navigator = (props: PropsType): ReactElement | null => {
   })
   const previousRouteKey = useRef<string | null | undefined>(null)
   const { fetchCities, fetchCategory, routeKey, routeName } = props
+  const cityCooperatingEnabled =
+    buildConfig().featureFlags.cityNotCooperatingTemplate && buildConfigAssets().cityNotCooperatingIcon
```

Comment:
maybe it would make sense to extract this one into a function in e.g. the cityCooperatingFooter and to consistently always check on both properties. Sometimes you only check for the template and sometimes only for the icon, which could be error prune imo and also feels very complicated.

Reply:
you are right, this was not really consistent. I adjusted it. CityCooperatingPages are only available when template and icon are set correctly, the Footer as a reusable component checks itself if icon is available.